BeerOverflowWindowsApp.UnitTests/TestData/BarProviderTestData.cs

```diff
@@ -2,14 +2,23 @@
 {
     public class BarProviderTestData
     {
+        public static string FacebookValidDataResponse = @"{""data"":[{""location"":{""latitude"":54,""longitude"":25},""name"":""TestPlace1"",""category_list"":[{""name"":""Bar""}]},{""location"":{""latitude"":54,""longitude"":25,},""name"":""TestPlace2"",""category_list"":[{""name"":""Bar""}]}]}";
+        public static string FourSquareValidDataResponse = @"{""response"":{""venues"":[{""name"":""TestPlace1"",""location"":{""lat"":54,""lng"":25,""distance"":0},""categories"":[{""id"":""4bf58dd8d48988d1c4941735"",""name"":""Restaurant""}]},{""name"":""TestPlace2"",""location"":{""lat"":54,""lng"":25,""distance"":0},""categories"":[{""id"":""4bf58dd8d48988d1c4941735"",""name"":""Restaurant""}]}]}}";
+        public static string GoogleValidDataResponse = @"{""html_attributions"":[],""results"":[{""geometry"":{""location"":{""lat"":54,""lng"":25}},""name"":""TestPlace1""},{""geometry"":{""location"":{""lat"":54,""lng"":25}},""name"":""TestPlace2""}],}";
+        public static string[] TripAdvisorValidDataResponse = {
+            @"{""data"":[{""location_id"":""1"",""name"":""TestPlace1"",""distance"":""0"",},{""location_id"":""2"",""name"":""TestPlace2"",""distance"":""0"",}]}",
+            @"{""data"":[]}",
+            @"{""latitude"":""54"",""location_id"":""1"",""longitude"":""25"",""category"":{""name"":""restaurant""},""subcategory"":[{""name"":""sit_down""}]}",
+            @"{""latitude"":""54"",""attraction_types"":[{""name"":""bar/ clubs""}],""location_id"":""2"",""longitude"":""25"",""groups"":[{""name"":""Nightlife"",""categories"":[{""name"":""Bars & Clubs""}]}],""category"":{""name"":""attraction""},""subcategory"":[{""name"":""nightlife""}]}"
+        };
```

Comment:
Why are there many cases for TripAdvisor and only one for each of the others?

Reply:
For TripAdvisor, we make additional requests, for each place, as we don't get its coordinates with the initial requests. We also make a request for each category.

So The first two responses are the initial ones. One for restaurants and second for attractions. The first one returns two places, the second is empty. So we need two more responses, for the collected place information.